src/ipc/mem/PageStack.h

```diff
@@ -45,39 +83,36 @@ class PageStack
     bool pageIdIsValid(const PageId &page) const;
 
     /// total shared memory size required to share
-    static size_t SharedMemorySize(const uint32_t aPoolId, const unsigned int capacity, const size_t pageSize);
+    static size_t SharedMemorySize(const uint32_t aPoolId, const PageCount capacity, const size_t pageSize);
     size_t sharedMemorySize() const;
 
     /// shared memory size required only by PageStack, excluding
     /// shared counters and page data
-    static size_t StackSize(const unsigned int capacity);
+    static size_t StackSize(const PageCount capacity);
     size_t stackSize() const;
 
     /// \returns the number of padding bytes to align PagePool::theLevels array
     static size_t LevelsPaddingSize(const unsigned int capacity);
```

Comment:
The type here should probably be using PageCount like all other capacity_ related items.

Reply:
Done.